src/lib/storage.ts

```diff
@@ -141,8 +141,10 @@ class Storage implements IStorageHandler {
           if (isOpen || err.status !== HTTP_STATUS.NOT_FOUND) {
             reject(err);
           }
-          // local reported 404
-          localStream.abort();
+          // local reported 404 or request was aborted already
+          if (localStream && localStream.abort) {
+            localStream.abort();
+          }
```

Comment:
It seems you try to solve a raising issue here - the localStream could be null, because of another error callback just happens? But seems to check localStream.abort is not necessary, it should be a function as always?

```
# Change to?
if (localStream) {
    localStream.abort();
    localStream = null;
}
resolve(false);
```

Reply:
My understanding is that localStream is already aborted here due to the call at line 154. 

True, I will simplify as suggested.